pkg/sign/signer.go

```diff
@@ -45,7 +45,7 @@ type BundleOptions struct {
 	// Optional list of Rekor instances to get transparency log entry from.
 	//
 	// Supports hashedrekord and dsse entry types.
-	Rekors []*Rekor
+	Transparencies []Transparency
```

Comment:
WDYT about TransparencyLog(s)? Or RekorLogs? I don’t think using Rekor is a problem, because this implementation only works with the Rekor API.

Reply:
TransparencyLogs is what I would prefer, it's used in a few other places too.